docs/notebooks/doc2vec-wikipedia.ipynb

```diff
@@ -158,8 +158,9 @@
    },
    "outputs": [],
    "source": [
+    "pres = Doc2VecVocab()\n",
     "pre = Doc2Vec(min_count=0)\n",
-    "pre.scan_vocab(documents)"
+    "pres.scan_vocab(documents)"
```

Comment:
why? this isn't used.

Reply:
scan_vocab function in Doc2VecVocab class as in http://localhost:8889/edit/gensim/models/doc2vec.py
That is the reason I have created another object to call that class and dot operator to call the function scan_vocab. All other functions such as build_vocab,init_sims,etc functions are in Doc2Vec class.

Reply:
let's try again: `Doc2VecVocab` class not used and not related to actual instance of Doc2Vec -> shouldn't be here.

Reply:
okay!